CHANGELOG.md

```diff
@@ -0,0 +1,47 @@
+# Change log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/).
+
+## [Master (Unreleased)]
+- Now `mutest` Can Be Run Without Being In The Bundle. ([@mvz][]) [[#46](https://github.com/backus/mutest/pull/46/files) ([@mvz][])]
+
+## [0.0.4] - 2017-02-16
+### Fixed
+- Only Emit Restarg Body Mutations When Used [[#44](https://github.com/backus/mutest/pull/44/files) ([@dgollahon][])]
+
+### Removed
+- Less Strict Relation Mutations (`<` -> `<=`; `>` -> `>=`) [[#47](https://github.com/backus/mutest/pull/47/files) ([@backus][])]
```

Comment:
A relational operator is a more specific form of binary operator. https://en.wikipedia.org/wiki/Relational_operator